src/width.rs

```diff
@@ -1075,23 +1076,26 @@ where
 pub(crate) fn wrap_text(text: &str, width: usize, keep_words: bool) -> String {
     if width == 0 {
         String::new()
-    } else if keep_words {
-        split_keeping_words(text, width, "\n")
     } else {
-        split(text, width, "\n")
+        let (prefix, text, suffix) = osc_partition(text).unwrap_or(("", text, ""));
```

Comment:
I don't think it will work in general case.
I mean it will work in basic case.

But what if the string has 2 links?

```rust
    let data = [
        Distribution {
            name: format!(
                "{} 2 links in a string {}",
                format_osc8_hyperlink("https://www.debian.org/", "Debian"),
                format_osc8_hyperlink("https://www.wikipedia.org/", "Debian"),
            ),
            is_hyperlink: true,
        },
    ];
```

```
".--------------------------------.\n| name            | is_hyperlink |\n| \u{1b}]8;;https://www                | true         |\n| .debian.org/    |              |\n| \u{1b}\\Debian \u{1b}]8;; \u{1b}\\         |              |\n| 2 links in a    |              |\n| string  \u{1b}]8;;htt        |              |\n| ps://www.wikipe |              |\n| dia.org/\u{1b}\\Debian  |              |\n| \u{1b}]8;; \u{1b}\\                |              |\n'--------------------------------'"
```

Reply:
Addressed by 5950c102c2146c7b0d6f5b6ad78831c8ed5086d6